src/i2c.rs

```diff
@@ -277,6 +275,12 @@ fn calculate_timing(
     an_filter: bool,
     dnf: u8,
 ) -> I2cTiming {
+    // We get an "unused" warning here when running `cargo test`. I haven't
+    // looked into it, but I suspect that is only used when `std` isn't
+    // available, which would explain that behavior.
```

Comment:
This problem is mentioned in the `micromath` docs: https://docs.rs/micromath/1.1.0/micromath/#unused-import-warnings-when-linking-std

Reply:
Thanks, @rfuest! I updated the comment to address that.